Automattic/vip
# Mount vip-config directly in wp

## Description

We load vip-config.php currently from `/wp/wp-content/vip-config/vip-config.php` however in live environments the vip-config folders is mapped directly to `/wp/vip-config`  (not under `wp-content`).

We should migrate the location to more closely match the prod environment. That way we can support other files too such as `client-sunrise.php` which [is loaded ](https://github.com/Automattic/vip-go-mu-plugins/blob/7294c3902d473c57c58e8505d4ad396de3470d0b/lib/sunrise/sunrise.php#L100-L106)from  `/wp/vip-config/client-sunrise.php`

This is the second part of the change where we change mount location

This should only be merged **AFTER** https://github.com/Automattic/vip-container-images/pull/516

## Steps to Test

1) Create new dev-env 
2) Attach shell to php contianer and see that vip-config is loaded under `/wp/vip-config` and everything still works.
